src/NzbDrone.Common/Cloud/LidarrCloudRequestBuilder.cs

```diff
@@ -16,7 +16,7 @@ public LidarrCloudRequestBuilder()
             Services = new HttpRequestBuilder("https://services.lidarr.audio/v1/")
                 .CreateFactory();
 
-            Search = new HttpRequestBuilder("https://api.lidarr.audio/api/v0.3/{route}")
+            Search = new HttpRequestBuilder("http://localhost:5001/{route}")
```

Comment:
This would be bumped to 0.4 when metadata changes are merged. You can set a custom metadata server in settings FYI, instead of changing it in code.

Reply:
Sorry, didn't mean to commit that. Will remove it.